src/content/blog/2025-10-23-redesigned-publisher-tools-technical.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "How the Redesigned Publisher Tools Work: A Technical Dive"
+description: "A technical exploration of the architecture details behind the redesigned Publisher Tools"
+date: 2025-10-23
+slug: redesigned-publisher-tools-technical-dive
+authors:
+  - Darian Avasan
+author_urls:
+  - https://www.linkedin.com/in/darian-avasan/
+tags:
+  - Interledger
+  - Web Monetization
+  - Publisher Tools
+  - Technical
+  - Architecture
+---
+
+If you haven’t heard of Publisher Tools yet, you’re missing out on a simple way to monetize your content! Don’t worry, you can catch up on the details [here]. Today, we’ll focus on the redesign.
+
+The Publisher Tools have been completely redesigned with a focus on simplicity, but there’s a lot happening behind that simplicity. Let's find out how things work after the embed script tag gets added to a website.
+
+### From Script Tag to Rendered Component: The Complete Flow
+
+![Configuration flow](/developers/img/blog/2025-10-17/tools-flowpng.png)
+
+**Key flow steps:**
+
+1. Publisher customizes customize through the interface (the `frontend` module) → Configuration stored as JSON (keyed by wallet address + preset tag e.g., ($ilp.link/your-wallet)", "version1").
+2. Visitor loads page with embed script → Script loads from `cdn` module
+3. Script fetches configuration → `api` retrieves stored JSON and returns it
+4. Script renders the Tool → Instantiates Web Component from `components` with fetched configuration
+
+This system cleanly separates responsibilities: publishers customize through the interface (the `frontend` module), and visitors consume the configuration at WebPage runtime. The `frontend` and visitor-facing flow never interact directly, they're connected only through stored configuration.
```

Comment:
Can also mention the components you see in preview are exactly same as what your users will see, as we use web components.